File: python/Algorithm/quick_search.py
def quick_search(list):
    #if len = 0 || len == 1 just return list as result
    if(len(list) < 2):
        return list
    #set a initial pivot
    pivot = list[0]
    #fine element which is less than pivot and great then the pivot 
    less = []
    greater = []
    for element in list[1:]:
        if element <= pivot:
            less.append(element)
        else:
            greater.append(element)
    return quick_search(less) + [pivot] + quick_search(greater)
            

if __name__ == "__main__":
    print(quick_search([2,3,4,1,3,4,5,68,3,4,5,3]))

File: JavaScript/Algorithm/quick_search.js
function quick_search(list) {
    if(list.length < 2) return list;
    let pivot = list[0];
    let less = [];
    let greater = [];
    for(let i = 1 ;i < list.length; i++) {
        if(list[i] <= pivot) {
            less.push(list[i]);
        } else {
            greater.push(list[i]);
        }
    }
    return quick_search(less) + [pivot] + quick_search(greater);
}

console.log(quick_search([5,3,4,5,6,7,8,9,12,34,65,8,1]));